FILE packages/desktop/src-tauri/tauri.conf.json
RANGE {"path":"packages/desktop/src-tauri/tauri.conf.json","old_lines":[53,57],"new_lines":[53,57],"diff_block_lines":[5,11],"hunk_header":"@@ -53,5 +53,5 @@","substantive_changed_lines":2}
@@ -53,5 +53,5 @@
         "https://github.com/getpaseo/paseo/releases/latest/download/latest.json"
       ],
-      "pubkey": "dW50cnVzdGVkIGNvbW1lbnQ6IG1pbmlzaWduIHB1YmxpYyBrZXk6IERDNjVEMDZGMUQ2MDhFMTgKUldRWWptQWRiOUJsM040L3ovSXQ0ZlhneDM1QkhDaTMxKzVkVTVWdE1USUw3OERmeEZNbFQrRUoK"
+      "pubkey": "dW50cnVzdGVkIGNvbW1lbnQ6IG1pbmlzaWduIHB1YmxpYyBrZXk6IEFGNzE3OTlBNEVCM0U4ODgKUldTSTZMTk9tbmx4cnh6OStYMGxFelpnRVJwTklXMlpyM3FQTXBVbFc5NUpFNmZMbUdjRjkyN3gK"
     }
   }
